src/main/java/me/spartacus04/jext/listeners/JukeboxClickEvent.kt

```diff
@@ -30,9 +30,13 @@ internal class JukeboxClickEvent : JextListener() {
     }
 
     private fun defaultBehaviour(event: PlayerInteractEvent, block: Block) {
-        if(!INTEGRATIONS.hasJukeboxAccess(event.player, block)) {
-             event.isCancelled = true
-             return
+        if(INTEGRATIONS.hasJukeboxAccess(event.player, block)==false) {
+            event.isCancelled = true
+            return
+        }
+        if(!(event.player.hasPermission("jext.usejukebox")||event.player.isOp)){
```

Comment:
This check should be moved to the OnJukeboxInteract method, you can also remove the `|| event.player.isOp` since it should be managed by a permission manager and not the plugin